nativescript-core/ui/core/view/view.ios.ts

```diff
@@ -56,18 +56,22 @@ export class View extends ViewCommon implements ViewDefinition {
         return (this._privateFlags & PFLAG_FORCE_LAYOUT) === PFLAG_FORCE_LAYOUT;
     }
 
-    public requestLayout(): void {
-        super.requestLayout();
+    public requestLayout(calledFromChild?:boolean): void {
```

Comment:
Space is needed after the colon in parameter to match the tslint rules
```suggestion
    public requestLayout(calledFromChild?: boolean): void {
```